frontend/src/pages/BenchmarkPage/index.tsx

```diff
@@ -4,64 +4,34 @@ import { useHistory, useLocation } from "react-router-dom";
 import { BenchmarkTable } from "../../components";
 import { BenchmarkCards } from "../../components/BenchmarkCards";
 import { backendClient } from "../../clients";
-import { PageHeader } from "antd";
+import { BenchmarkConfig } from "../../clients/openapi";
 
 function useQuery() {
   const { search } = useLocation();
   return React.useMemo(() => new URLSearchParams(search), [search]);
 }
 
-async function getTasks() {
-  const allTasks = Array<string>();
-  const taskConfigs = await backendClient.tasksGet();
-  taskConfigs.map(({ tasks }, i) =>
-    tasks.map(({ name }) => allTasks.push(name))
-  );
-  return allTasks;
-}
-
-async function getBenchmarks() {
-  const allBenchmarks = Array<string>();
-  const benchmarkConfigs = await backendClient.benchmarkconfigsGet();
-  benchmarkConfigs.map(({ name }) => allBenchmarks.push(name));
-  return allBenchmarks;
-}
-
 export function BenchmarkPage() {
   const history = useHistory();
   const query = useQuery();
-  const name = query.get("name") || undefined;
+  const id = query.get("id") || "";
 
-  const [items, setItems] = useState<string[]>([]);
+  const [items, setItems] = useState<BenchmarkConfig[]>([]);
 
   useEffect(() => {
     async function fetchItems() {
-      if (name === "Global") {
-        setItems(await getTasks());
-      } else if (name == null) {
-        setItems(await getBenchmarks());
-      }
+      setItems(await backendClient.benchmarkconfigsGet(id));
     }
     fetchItems();
-  }, [name, history]);
+  }, [id, history]);
 
-  if (name === "Global" || name == null) {
+  if (id === "" || items.length !== 0) {
     return <BenchmarkCards items={items} subtitle="Select a benchmark" />;
   } else {
-    let title = "";
-    title = name;
-    let subTitle = "";
-    subTitle = `${name}`;
-
     return (
       <div>
-        <PageHeader
```

Comment:
Deleting this code gets rid of the back button on the GaoKao or Masakhaner page. I think a simple fix is to put a PageHeader with a back button in the return statement on line 132 of BenchmarkTable.

Reply:
That's a good point. I tried adding the back button, but it didn't look great aesthetically so in the end I removed it.